### ISSUE TITLE
question about a part of code

### ISSUE THREAD

**Issue by Powercoder64** [zhang-can/CoLA#7]
Hello,

Thanks for your great work.

I have a question:

Why do you use zeros as labels in the loss function and not the original labels?

I am talking about this part from NCE function:

`labels= torch.zeros(logits.shape[0], dtype=torch.long).cuda()`

**Comment by zhang-can** [comment]
Hi @Powercoder64 , 

Thanks for your interest in our work.

Following [MoCo code](https://github.com/facebookresearch/moco/blob/main/moco/builder.py#L155), we also use CE loss for implementation. Since we put positives at 0-th locations, the GT labels should be all 0. More details can be found in Algorithm 1 of [MoCo paper](https://arxiv.org/pdf/1911.05722.pdf).
